### PR TITLE
CLN: Parameterize test case

### PR DESCRIPTION
**Pull request by alimcmaster1**
- [y] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

File: pandas/tests/scalar/timestamp/test_unary_ops.py [hunk 1]
```diff
@@ -19,26 +19,16 @@ class TestTimestampUnaryOps(object):
...
-        expected = Timestamp('20130105')
+    @pytest.mark.parametrize('timestamp_input, round_freq, date', [
+        ('20130101 09:10:11', 'D', '20130101'),
+        ('20130101 19:10:11', 'D', '20130102'),
+        ('20130201 12:00:00', 'D', '20130202'),
+        ('20130104 12:00:00', 'D', '20130105'),
+    ])
+    def test_round_day_naive(self, timestamp_input, round_freq, date):
+        dt = Timestamp(timestamp_input)
+        result = dt.round(round_freq)
```
[the remark this answers is not in the document]
Comment (alimcmaster1) [reply pandas/tests/scalar/timestamp/test_unary_ops.py]: thanks for taking a look @WillAyd , turns out there was another similar test function in the same file so I combined them. What do you think?

[the remark this answers is not in the document]
Comment (WillAyd) [reply pandas/tests/scalar/timestamp/test_unary_ops.py]: Nice! Just one more minor nit looks like your test is called `test_test_round_frequencies` - can just change to `test_round_frequencies`

[the remark this answers is not in the document]
Comment (alimcmaster1) [reply pandas/tests/scalar/timestamp/test_unary_ops.py]: Ahh my bad.. thanks have done!

File: pandas/tests/scalar/timestamp/test_unary_ops.py [hunk 2]
```diff
@@ -19,26 +19,19 @@ class TestTimestampUnaryOps(object):
...
-
-        dt = Timestamp('20130201 12:00:00')
-        result = dt.round('D')
-        expected = Timestamp('20130202')
-        assert result == expected
-
-        dt = Timestamp('20130104 12:00:00')
-        result = dt.round('D')
-        expected = Timestamp('20130105')
+    @pytest.mark.parametrize('timestamp_input, round_freq, date', [
```
Comment (jreback) [inline pandas/tests/scalar/timestamp/test_unary_ops.py]: just call this ``timestamp, freq, expected`` for the args, otherwise lgtm.

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/tests/scalar/timestamp/test_unary_ops.py (hunk 2 above)
Comment (alimcmaster1) [reply pandas/tests/scalar/timestamp/test_unary_ops.py]: thanks @jreback updated

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
lgtm. ping on green.

**Comment by jreback** [comment]
thanks @alimcmaster1

**Pull request merged**